InventoryImportExport/etc/di.xml

```diff
@@ -31,6 +31,14 @@
     <type name="Magento\CatalogImportExport\Model\StockItemImporterInterface">
         <plugin name="importStockItemDataForSourceItem" type="Magento\InventoryImportExport\Plugin\Import\SourceItemImporter" />
     </type>
+    <!-- Prevent stock indexer to execute twice as it is run by default -->
+    <type name="Magento\CatalogImportExport\Model\Import\Product\StockProcessor">
+        <arguments>
+            <argument name="indexers" xsi:type="array">
+                <item name="cataloginventory_stock" xsi:type="null"/>
```

Comment:
Does it mean that we can delete `Magento\CatalogImportExport\Model\Import\Product\StockProcessor` at all?

Reply:
Here we prevent stock indexer to run before MSI stock handlers. We must not delete `Magento\CatalogImportExport\Model\Import\Product\StockProcessor` because it's required to Magento installation without MSI (e.g Magento CE only installation).

Reply:
branch updated with `1.1-develop`

Reply:
Well, but I as far I can see, `Magento\CatalogImportExport\Model\Import\Product\StockProcessor` is used in only one place: `Magento\CatalogImportExport\Model\Import\Product`, and the only argument is passed is [there](https://github.com/magento/magento2/blob/73ed5cc4868803ed6a84cb66283566697c579acd/app/code/Magento/CatalogImportExport/etc/di.xml#L22)

So, now, its `$indexers` list is always empty, and it always does nothing?)

Is there a place it is used I failed to find? Maybe, it's my bad)

Reply:
That's correct. That's the intended behavior. we don't want `Magento\CatalogImportExport\Model\Import\Product\StockProcessor` to execute  `cataloginventory_stock` when MSI is installed. Because MSI run `cataloginventory_stock` afterwards. Without MSI this processor will execute `cataloginventory_stock` as expected (e.g Magento CE only installation)